File: existant/gestFormationWeb/gestion_formation_web/dto/DtoTheme.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using gestion_formation_web.dao;

namespace gestion_formation_web.dto
{
    public class DtoTheme
    {
        public static IEnumerable<theme> Get()
        {   
            return DaoTheme.Get();
        }
    }
}

File: Gestionnaire de formation/src/main/java/fr/esic/solutec/dto/DtoTheme.java
package fr.esic.solutec.dto;

import java.util.List;


import fr.esic.solutec.dao.DaoTheme;
import fr.esic.solutec.entities.Theme;

public class DtoTheme {

	
	public static Theme getTheme(int idTheme) {
		
		return DaoTheme.getTheme(idTheme);
	}
	
	public static List<Theme> getListTheme() {
		
		return DaoTheme.getListTheme();
	}
	
	
	
	
}
